Gemfile

```diff
@@ -8,6 +8,8 @@ gem 'rubocop', '>= 1.0', '< 2.0'
 # Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
 gem 'rails', '~> 7.0.3'
 
+gem 'ffi', github: 'ffi/ffi', submodules: true
```

Comment:
[ Optional Suggestion ]

- [ ] I face issue running the app with this `gem 'ffi', github: 'ffi/ffi', submodules: true` setup. I had to remove the Github to the end before it could run. `gem 'ffi'`
- [ ] You could have used this one `gem 'ffi', '~> 1.15', '>= 1.15.5'`. Try it and see 👍 Just a suggestion though.

Reply:
Thanks for the advice 👍
I will definitely check  🏃‍♂️🌟
This was good research for me 🛠📌